google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -153,9 +148,20 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    configuration.setCloudProjectName(projectSelector.getText());
-    configuration.setCloudSdkExecutablePath(
-        CloudSdkUtil.toExecutablePath(cloudSdkDirectoryField.getText()));
+    if (StringUtil.isEmpty(cloudSdkDirectoryField.getText())
```

Comment:
In case you are wondering why  the duplicated message in the first screenshot - the red warning underneath the inputs is dynamic validation which gets updated whenever the user updates the field. The block at the bottom is when the user tries to save the entire panel.